mcp-tools/maos-mcp-hub/lib/confluence/client.py

```
@@ -0,0 +1,231 @@
+            method="POST",
+            url=f"{self.base_url}/footer-comments",
+            provider=self._provider_name,
+            auth_hint=self._auth_hint,
+            auth_kwargs=self._auth_kwargs,
+            json={"pageId": page_id, "body": {"representation": "storage", "value": body}},
+            timeout=30.0,
+            limiter=self.rate_limiter,
+            max_retries=0,
+        )
+
+    async def create_inline_comment(self, page_id: str, body: str, inline_comment_properties: dict = None) -> dict:
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)